aws/logs_monitoring/README.md

```diff
@@ -39,6 +48,12 @@ Note: you can find the installed Forwarder under the stack's "Resources" tab.
 
 Note: It's recommended to adjust forwarder settings through CloudFormation rather than directly editing the Lambda function. The description of settings can be found in the [template.yaml](template.yaml) and the CloudFormation stack creation user interface when you launch the stack. Feel free to submit a pull request to make additional settings adjustable through the template.
 
+## Settings
+
+To view all the adjustable settings of the forwarder, click "Launch Stack" from the [Installation](#installation) section and you will be prompted with a CloudFormation user interface with all the adjustable settings (you do not have to complete the installation).
```

Comment:
I think this is better than re-documenting what's already in the template.yaml again in the README. The CloudFormation interface actually provides a pretty nice view of settings, given the dropdown, ordering, and grouping features. Based on the future user feedback, we may decide to actually list out of the parameters in a table under this section.